src/Cursor.js

```diff
@@ -37,8 +37,10 @@ function update(cmp, path, operation, nextValue) {
   var nextState;
 
   if (path.length > 0) {
+    var q = cmp._reactInternalInstance._pendingStateQueue;
```

Comment:
RIght, let me rephrase: does React .13 include improvements to the React API that make it no longer necessary for cursors to access the pending state?